tests/foreman/cli/test_contentview.py

```diff
@@ -1231,6 +1233,57 @@ def test_positive_add_rh_repo_by_id_and_create_filter(self):
             'name': 'walgrind',
         })
 
+    @run_in_one_thread
+    @tier3
+    @upgrade
+    def test_positive_add_module_stream_filter_rule(self):
+        """Associate module stream content to a content view and create filter rule
+
+        :id: 8186a4b2-1c11-11ea-99cf-d46d6dd3b5b2
+
+        :setup: Sync custom yum repo content
+
+        :steps: Assure filter(s) applied to associated content
+
+        :expectedresults: Filter rule should get applied to Filter
+
+        :CaseImportance: Low
+
+        :CaseLevel: Integration
+
+        """
+        filter_name = gen_string('alpha')
+        repo_name = gen_string('alpha')
+        org = entities.Organization().create()
+        create_sync_custom_repo(
+            org.id,
+            repo_name=repo_name,
+            repo_url=CUSTOM_MODULE_STREAM_REPO_2)
+        repo = entities.Repository(name=repo_name).search(
+            query={'organization_id': org.id})[0]
+        content_view = entities.ContentView(organization=org.id, repository=[repo]).create()
+        walrus_stream = ModuleStream.list({'search': "name=walrus, stream=5.21"})[0]
+        content_view = ContentView.info({u'id': content_view.id})
+
+        self.assertEqual(
+            content_view['yum-repositories'][0]['name'],
+            repo.name,
+            'Repo was not associated to CV',
+        )
+        content_view_filter = ContentView.filter.create({
+            'content-view-id': content_view['id'],
+            'inclusion': 'true',
+            'name': filter_name,
+            'type': 'modulemd',
+        })
+        content_view_filter_rule = ContentView.filter.rule.create({
+            'content-view-filter': filter_name,
+            'content-view-id': content_view['id'],
+            'module-stream-ids': walrus_stream['id'],
+        })
+        filter_info = ContentView.filter.info({'id': content_view_filter['filter-id']})
+        assert filter_info['rules'][0]['id'] == content_view_filter_rule['rule-id']
+
```

Comment:
Just a thought , Do you think a content_view publish and check filtered packages in new version does add value here ?

Reply:
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_contentview.py#L2650 test already doing it, I just wanted to see if the hammer commands work fine creating content view filter rules